src/main/java/at/hannibal2/skyhanni/features/inventory/SkyblockGuideHighlightFeature.kt

```diff
@@ -0,0 +1,159 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryFullyOpenedEvent
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.ItemUtils.name
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.highlight
+import at.hannibal2.skyhanni.utils.StringUtils.anyMatches
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import org.intellij.lang.annotations.Language
+
+val group = RepoPattern.group("skyblockguide.highlight")
+
+private const val keyPrefixInventory = "inventory"
+private const val keyPrefixCondition = "condition"
+
+class SkyblockGuideHighlightFeature private constructor(
```

Comment:
This looks like a good use case for an enum.

Reply:
I can't get it to work. Because it has multiple constructor + values in the companion object (which isn't initialized yet)